CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+## [0.9.1](https://github.com/haskell-nix/hnix/compare/0.9.0...0.9.1) (2020-07-13)
+
+* Previous release was `0.9.0`. In new release:
+
```

Comment:
```suggestion
```

I thought the reference to the previous version in the changelog for v0.9.0 was a special case since it was the first changelog entry.

I might be mistaken though!

Reply:
Yup, it feels superfluous, removed.